packages/projects-docs/pages/learn/devboxes/synced-templates.mdx

```diff
@@ -20,27 +20,38 @@ Synced templates are [templates](https://codesandbox.io/docs/learn/devboxes/temp
 
 CodeSandbox also has [Repositories](https://codesandbox.io/docs/learn/repositories/overview), which are also imported from GitHub repositories. However, there is a key difference between synced templates and repositories.
 
-Repositories are meant for working on a repository. We introduce a workflow to create new branches and pull requests. Synced templates allow you to create a sandbox template from a repository (or folder), and are meant to be used as a template for new sandboxes (or for sharing as an example with others).
+Repositories are meant for working on a repository. We introduce a workflow to create new branches and pull requests. Synced templates allow you to create a Devbox/Sandbox template from a repository (or folder), and are meant to be used as a template for new projects (or for sharing as an example with others).
 
 A good rule of thumb is that if you want to work on a repository, you should import a repository. If you want to create a template from a repository, you should create a synced template from a GitHub repository.
 
 ## Creating a synced template
 
-A synced template URL is essentially the same as the GitHub URL, but `github.com` is replaced with `codesandbox.io/p/devbox/github`. Because of this, you can go to your GitHub repository, and replace `github.com` with `codesandbox.io/p/devbox/github` to create a synced template. We have also introduced three helpers to do this more easily:
+A synced template URL is essentially the same as the GitHub URL, but `github.com` is replaced with a CodeSandbox path.
 
-https://codesandbox.io/p/devbox/github/withastro/astro/main/examples/blog
+We use some heuristics to detect if a given repository should be opened as a synced Sandbox or a synced Devbox.
```

Comment:
I find this confusing. Are you explaining how to created a sandbox or devbox through a url or are you provided an explanation of the urls for synced projects.

Reply:
For this intro section, I recommend being a lot more explicit on the what the process is and who the agent is (or can be)

Example: 
```
A synced template runs code from a GitHub branch in CodeSandbox and keeps the changes synced with remote. A synced template can either be a Sandbox or a Devbox. You can tell the difference by the way the url is constructed. 

- a synced template as a Devbox will look something like this: `codesandbox.io/p/devbox/github`.
-  a synced template as a Sandbox will look something like this:  `codesandbox.io/p/sandbox/github`.

Synced templates can be created directly from the Github URL or through our GitHub importer tool
```